### DISCUSSION TITLE
How To Add SSL?

### DISCUSSION THREAD

**Discussion by auzty** [thedev-id/thedev.id#392] [Q&A]
I already add domain from firebase hosting based, but when I access it, the certificate are invalid (because using ssl from firebaseappp)
, and I see the other is using vercel or netifly, but no issue and can add letsencrypt certificate.
what I must do? 
thank you..

**Comment by ginanjardwiprasetyo** [comment]
```
NET::ERR_CERT_COMMON_NAME_INVALID

gin-profile.thedev.id normally uses encryption to protect your information. When Chrome tried to connect to gin-profile.thedev.id this time, the website sent back unusual and incorrect credentials. This may happen when an attacker is trying to pretend to be gin-profile.thedev.id or a Wi-Fi sign-in screen has interrupted the connection. Your information is still secure because Chrome stopped the connection before any data was exchanged.

You cannot visit gin-profile.thedev.id right now because the website uses HSTS. Network errors and attacks are usually temporary, so this page will probably work later.
```
@fransallen I still don't understand, can you help?

**Reply by ginanjardwiprasetyo** [reply]
@ahmadhayyan maybe can help?

**Reply by ahmadhayyan** [reply]
Have you set the custom domain on your GitHub page (or host)?
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site

**Reply by ginanjardwiprasetyo** [reply]
@auzty @ahmadhayyan Ah, thats right, in other case I have try to add file CNAME manually that include my subdomain like *.thedev.id, just it and viola "https" live. Thanks for your help @ahmadhayyan
